src/Components/ExposedLogExplorationPage/v1/ExposedLogExplorationPage.tsx

```diff
@@ -0,0 +1,76 @@
+import { AdHocVariableFilter, TimeRange, dateTimeParse } from "@grafana/data";
+import { SceneTimeRange } from '@grafana/scenes';
+import { IndexScene } from "Components/IndexScene/IndexScene";
+import { isEqual } from "lodash";
+import React, { useEffect, useMemo } from "react";
+
+const DEFAULT_TIME_RANGE = { from: 'now-15m', to: 'now' };
+
+interface Props {
+  initialFilters?: AdHocVariableFilter[];
+  initialDS?: string;
+  initialTimeRange?: TimeRange;
+  onTimeRangeChange?: (timeRange: TimeRange) => void;
+}
+
+export function ExposedLogExplorationPage({ initialFilters, initialDS, initialTimeRange, onTimeRangeChange }: Props) {
+  const [isInitialized, setIsInitialized] = React.useState(false);
+  // Must memoize the top-level scene or any route change will re-instantiate all the scene classes
+  const scene = useMemo(
+    () =>
+      new IndexScene({
+        $timeRange: new SceneTimeRange(
+          initialTimeRange
+            ? {
+                from:
+                  typeof initialTimeRange.raw.from === 'string'
+                    ? initialTimeRange.raw.from
+                    : initialTimeRange.raw.from.toISOString(),
+                to:
+                  typeof initialTimeRange.raw.to === 'string'
+                    ? initialTimeRange.raw.to
+                    : initialTimeRange.raw.to.toISOString(),
+              }
+            : DEFAULT_TIME_RANGE
+        ),
+        $behaviors: [
+          (scene: IndexScene) => {
+            const unsubscribable = scene.state.$timeRange?.subscribeToState((newState, oldState) => {
+              if (!isEqual(newState, oldState)) {
+                const timeRange: TimeRange = {
+                  from: dateTimeParse(newState.from),
+                  to: dateTimeParse(newState.to),
+                  raw: {
+                    from: newState.from,
+                    to: newState.to,
+                  },
+                };
+                onTimeRangeChange?.(timeRange);
+              }
+            });
+
+            return () => {
+              unsubscribable?.unsubscribe();
+            };
+          },
+        ],
+        initialFilters,
+        initialDS,
```

Comment:
What happens when this is empty when optional?

Reply:
this is a really good question, and I would say a more global one.
Explore logs app has 2 modes: `'service_selection' | 'service_details'`. Not sure I can imagine the situation where we need to "embed" and start from `service_selection` view. So my initial idea was to use only `service_details` view with a particular `service_name` filter filled (this is a condition to have this mode opened, otherwise explore logs app redirects to `service_selection`). I am thinking about sticking to this (and this made me think about some other changes - like I need to disable removing `service_name` filter for example) wdyt @jarben ?

so answering your question: 
1. If `initialFilters` are empty and the mode is `service_details` currently it just redirects to explore logs app with `service_selection` view enabled.
2. If `initialDS` is empty it applies the default one.

Reply:
Interesting, thanks for all the details @kozhuhds! So the question is whether to make the extension generic or specific to a service?
- If generic, making the `mode` as a param would make sense so users can explicitly choose `service_selection` if that's what they want.  
- if service-specific, what if we rename the extension to `explore-service-logs/v1` and make the `service_name ` param required? (we can also make an extension such as `explore-services`)